### PR TITLE
Added touchscreen

### PR DESCRIPTION
**Pull request by seekbirdy**

### CODE REVIEW

File: partcad.yaml [hunk 1]
```diff
@@ -30,6 +30,22 @@ parts:
       corner-bottom-back-right: [[85, 56, 0], [0, 0, 1], 180]
       corner-bottom-front-left: [[0, 0, 0], [0, 0, 1], 0]
       corner-bottom-front-right: [[85, 0, 0], [0, 0, 1], 90]
+  rpi-touchscreen:
```
Comment (openvmp) [inline partcad.yaml:33]: Does it make sense to call it "shields/rpi-touchscreen" and drop the "path" attribute?

**Review by openvmp — APPROVED** [review]

File: partcad.yaml (hunk 1 above)
Comment (openvmp) [reply partcad.yaml:33]: "shields/touchscreen"

Comment (seekbirdy) [reply partcad.yaml:33]: I agree we can minimize the naming (it is clear that it is RPI common touchscreen), but why then not to drop "shields" too? Also, when we use this outside (in other package), how do we easily see the context (whose touchscreen it is among others?)

Comment (seekbirdy) [reply partcad.yaml:33]: I suggest to leave it as it is now and decide if we want to refactor this later

**Pull request merged**